### PR TITLE
[Changelog] Removing PRs from changelog

### PR DESCRIPTION
**Pull request by JPrause**
[skip ci]

### CODE REVIEW

**Comment by JPrause** [comment]
@miq-bot add_label documentation

**Comment by JPrause** [comment]
@miq-bot add_label hammer/yes

**Comment by JPrause** [comment]
@simaishi FYI

**Comment by simaishi** [comment]
@JPrause You're doing the other way around... we should be removing some PRs from the original Hammer-5 changelog.

**Comment by mturley** [comment]
I think we should mention that the conversion hosts settings feature was added, even though we shouldn't show all these intermediate steps towards adding it. @JPrause maybe we can add a line to the changelog linking to https://github.com/ManageIQ/manageiq-v2v/pull/922 but instead of saying "Enable (un-hide)" we should just call it "Add Conversion Hosts tab to Settings page". I'll put links to the relevant PRs in that PR for anyone curious enough to click through.

Sorry for the extra churn :) I'll start using `changelog/yes` and `changelog/no` in the future.

**Comment by simaishi** [comment]
> we can add a line to the changelog linking to #922 but instead of saying "Enable (un-hide)" we should just call it "Add Conversion Hosts tab to Settings page"

True, but either way, that's for `hammer-6`!

**Comment by mturley** [comment]
Oh right! Duh. Ignore me.

**Comment by simaishi** [comment]
@JPrause Anything that was backported this week shouldn't be included, so you can drop the first commit https://github.com/ManageIQ/manageiq-v2v/pull/946/commits/be458d840ed9369264957d0ca5bdc454d8438306 from this PR.

@mturley Can you please review the rest of the changes (the removed PRs)?

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,41 +8,21 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 ## Hammer-5
 
 ### Added
-- Use conversion_hosts API for RHV target cluster warnings instead of host CF tags [(#847)](https://github.com/ManageIQ/manageiq-v2v/pull/847)
```
Comment (mturley) [inline CHANGELOG.md]: This one can stay in, it's not related to the new Conversion Host Enablement settings. I think it's the only conversion host related PR that should stay.

**Comment by simaishi** [comment]
@JPrause Please fix the PR title as well as the git commit message as this PR isn't "adding missing PRs" but "removing extra PRs" 😄

**Comment by JPrause** [comment]
No worries,...although I've been having git rebase nightmares the past few nights. :+1:

**Comment by simaishi** [comment]
Hammer backport details:

```
$ git log -1
commit 5a839aad6f9bc00693da7e72232f084c57a4e35f
Author: Mike Turley <mike.turley@alum.cs.umass.edu>
Date:   Fri Apr 26 13:54:29 2019 -0400

    Merge pull request #946 from JPrause/hammer5.1_cl
    
    [Changelog] Removing PRs from changelog
    
    (cherry picked from commit a031eea9a38d2390436de1c227d3bf66296a733b)
```

**Pull request merged**